apps/cowswap-frontend/src/modules/orderProgressBar/hooks/useOrderProgressBarProps.ts

```diff
@@ -289,6 +289,10 @@ function useCountdownStartUpdater(
   const setCountdown = useSetExecutingOrderCountdownCallback()
 
   useEffect(() => {
+    if (!orderId) {
```

Comment:
```
function useCountdownStartUpdater(
  orderId: string,
  countdown: OrderProgressBarState['countdown'],
  backendApiStatus: OrderProgressBarState['backendApiStatus'],
  shouldDisableCountdown: boolean,
)
```

the function accepts orderId only as a string, is it possible to add " | undefined" in types for better understanding that it could be undefined ?